index.js

```diff
@@ -35,15 +36,31 @@ const argv = yargs
     .alias('help', 'h')
     .argv;
 
+
 const cleanOutput = packages => Object.entries(packages).map(([key, value]) => {
-    const { path, licenseFile, ...rest } = value;
-    const validInfo = {
-      package_name: key,
-      ...rest,
-    };
+  const { path, licenseFile, ...rest } = value;
+  const validInfo = {
+    package_name: key,
+    ...rest,
+  };
+
+  return validInfo;
+});
+
+
+const createLicenseReport = ({ report }) => {
+    const compiled = _.template(
+      '|   Library   |  Version  |  License |    Repository   |\n|---|---|---|---|\n<% _.forEach(report, elem => { const r = /(.*)@(.*)/.exec(elem.package_name); %>| <%- r[1] %> | <%- r[2] %> | <%- elem.licenses %> | <%- elem.repository %> |\n <% }); %>');
 
-    return validInfo;
+  const output = compiled({ 'report': report });
+  fs.writeFileSync(
+    `${argv.outputFileName}.md`,
+    output, error => {
+      if (error) throw error;
   });
+  console.info(`${argv.outputFileName}.md created!`);
+};
+
```

Comment:
In the future we could refactor this and move it to a utils folder, as we'll have more auxiliary methods